.azure-pipelines-templates/deploy_aci.yml

```diff
@@ -16,7 +16,20 @@ jobs:
 
       - script: |
           set -ex
-          python3.8 scripts/azure_deployment/arm_template.py deploy aci --subscription-id $(CCF_AZURE_SUBSCRIPTION_ID) --resource-group ccf-aci --aci-type dynamic-agent --deployment-name ci-$(Build.BuildNumber) --aci-ssh-keys "$(generate_ssh_key.hostPubKey)" > ~/aci_ips
+          RETRIES=100
```

Comment:
That seems like a lot?

Reply:
Yes, I'm thinking of keeping it high until we have a more realistic time range for how long building CCF as part of building the image takes